src/hooks/useLocalStorageItem.test.ts

```diff
@@ -22,8 +22,37 @@ describe(useLocalStorageItem, () => {
     localStorage.setItem(key, `"${updatedValue}"`);
 
     const hook = renderHook(() => useLocalStorageItem(key, blankValue));
+    
+    // function workWithUnsafeData(data: unknown) {
+    //   data.toUpperCase();
 
-    await act(async () => {});
+
+    //   if (typeof data === 'string') {
+    //     data.toUpperCase();
+    //   }
```

Comment:
...unless you _narrow_ it first.

https://www.typescriptlang.org/docs/handbook/2/narrowing.html